test/modules/fundingManager/extensions/FM_EXT_TokenVault_v1.t.sol

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "forge-std/Test.sol";
+
+import "forge-std/console.sol";
+
+import {IERC165} from "@oz/utils/introspection/IERC165.sol";
+
+import {Clones} from "@oz/proxy/Clones.sol";
+
+import "@oz/utils/Strings.sol";
+
+import {
+    ModuleTest,
+    IModule_v1,
+    IOrchestrator_v1
+} from "test/modules/ModuleTest.sol";
+
+// SuT
+import {
+    FM_EXT_TokenVault_v1,
+    IFM_EXT_TokenVault_v1
+} from "@fm/extensions/FM_EXT_TokenVault_v1.sol";
+
+import {Module_v1, IModule_v1} from "src/modules/base/Module_v1.sol";
+
+import {
+    IFundingManager_v1,
+    FundingManagerV1Mock
+} from "test/utils/mocks/modules/FundingManagerV1Mock.sol";
+import {ERC20Mock} from "test/utils/mocks/ERC20Mock.sol";
+// Errors
+import {OZErrors} from "test/utils/errors/OZErrors.sol";
+
+contract FM_EXT_TokenVault_v1Test is ModuleTest {
```

Comment:
We don't have any Gherkin in the file.

I've just checked if ChatGPT could help with that and it's pretty good, which could safe you some time. Example:
```
/* Test validAmount() modifier
    ├── Given the vault contains a specific amount of tokens
    │   └── And the amount is equal to 0
    │       └── When the withdraw() function is called
    │           └── Then the transaction should revert with Module__FM_EXT_TokenVault__InvalidAmount error
    └── Given the amount is greater than 0
        └── When the withdraw() function is called
            └── Then the transaction should succeed
*/
```

Reply:
I test both conditions here
The first one in the test itself
The second in the modifier in position / the modifier test itself

Reply:
I will be adding some gherkin to your tests. Will prob do that somewhere later today/tomorrow